iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/model/ProjectOrganization.java

```diff
@@ -0,0 +1,54 @@
+package eu.dnetlib.iis.wf.affmatching.model;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Paired project and organization identifiers.
+ * 
+ * @author mhorst
+ *
+ */
+public class ProjectOrganization {
```

Comment:
All model objects renamed by adding `AffMatch` prefix and moved to `eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model`.